packages/editor/src/client/panel/imageEditPanel.tsx

```diff
@@ -274,6 +276,7 @@ export function ImagedEditPanel({id, file, onClose, onSave}: ImageEditPanelProps
                 />
               )}
             </Panel>
+            {resizeImageHint(file!)}
```

Comment:
This leads to an error if i click an Image in the "Image Library" and the `imageEditPanel` opens.

Error:
![image](https://user-images.githubusercontent.com/2340416/141457807-327e12cf-1a93-41b9-bf80-56570ab28e25.png)

My suggestion is to display the hint about image resizing always BUT again not in the `imageEditPanel`. This information is needed at the time of upload which would be in the `imageUploadPanel`. Right next to the `<FileDropInput ... />` component.

Reply:
The time of upload is not in the imageUploadPanel ;) But I moved it anyway to the imageUploadPanel. Looks good for me as well.